packages/myst-frontmatter/src/frontmatter/frontmatter.spec.ts

```diff
@@ -27,7 +27,7 @@ import {
 const TEST_AUTHOR: Author = {
   userId: '',
   name: 'test user',
-  orcid: 'https://orcid.org/0000-0000-0000-0000',
+  orcid: '0000-0000-0000-0000',
```

Comment:
Looking at the `orcid` library, looks like the original URL input would still be accepted, and just coerced to the id itself. Just changed the test since it compares before and after to the same object, so coercion breaks the tests.

Reply:
Yep, I am starting to build out more of the tests in YAML, which are a bit easier to understand and capture more of the usecases I think. (e.g. author/authors).